src/components/WalletActionBar/index.tsx

```diff
@@ -71,6 +78,10 @@ export class WalletActionBar extends React.Component<WalletActionBarProps> {
       }, 2000);
     }
   };
+
+  private handleRemove = () => {
```

Comment:
another way of doing this is to define handler in the `inject` section like this https://github.com/LykkeCity/LykkeExchangeWeb/commit/46afa83bcb45d91b89cafb76084ddc22ebc47a42#diff-982702be75a4d0aff3103871c68a0141R90

plz have a look and lmk what do you think

Reply:
I like this handler, implemented.